app/instance-initializers/violations-helper.js

```diff
@@ -0,0 +1,11 @@
+import ViolationsHelper from 'ember-a11y-testing/utils/violations-helper';
+
+export function initialize(application) {
+  window.violationsHelper = new ViolationsHelper();
```

Comment:
Probably. :) 

My concern is exposing the global in production builds, which might not happen. But that's what I'm trying to guard against